src/middlewares/verifyUser.js

```diff
@@ -0,0 +1,16 @@
+import models from '../models';
+import { errorResponse } from '../utils';
+
+export default async (req, res, next) => {
+  const { userId } = req;
+  try {
+    const { dataValues: user } = await models.Users.findOne({ where: { id: userId } });
+    if (!user) {
+      return errorResponse(res, 400, 'Invalid Token');
```

Comment:
I agree. Sounds more accurate. the reason for saying `Invalid token` was because the token was supposed to be linked to a registered user. So, if the `userId` it decodes into does not get us a user from the database, then the token must be bad. But like I said, makes more sense to simply say `user not found`. Thanks.

Reply:
OKay. Now I get. You're right. I guess I was thinking in terms of failures beyond the code. Like Network calls etc.